### ISSUE TITLE
TEAMS PASSWORD NOT REMEMBERED

### ISSUE THREAD

**Issue by kalusm** [ramboxapp/community-edition#2276]
AFTER EVERY RESTART OF RAMBOX I NEED TO REENTER CREDENTIALS FOR MICROSOFT TEAMS!!!! HUGELY ANNOYING CONSIDERING THE FACT THAT OUTLOOK365 THAT IS USING EXACTLY SAME ACCOUNT WORKS FINE!!!!

	
	-
	> Rambox 0.6.4
	> Electron 3.1.4
	> linux x64 5.0.11-200.fc29.x86_64

**Comment by Corchoneitor** [comment]
Hi @kalusm.

I tested in my computer and the service remembers I logged in before.

Are you sure you checked the "Keep me signed in" and "Stay signed in?" options? Most importantly the second one, which is not that common.

**Comment by kalusm** [comment]
I have several other services in Teams: Outlook365, Slack, Skype.

All are working as well except Teams.

Juan Manuel Gonzalez Alama píše v Čt 23. 05. 2019 v 15:12 -0700:

Hi @kalusm<https://github.com/kalusm>.

I tested in my computer and the service remembers I logged in before.

Are you sure you checked the "Keep me signed in" and "Stay signed in?" options? Most importantly the second one, which is not that common.
